templates/map.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if map_token is defined and map_token != 'your_mapbox_access_token' %}
```

Comment:
I would prefer to not test the value of the token. If someone forget to change the default token value, it's up to Mapbox to report the error, not to us. Moreover, there's no error message to tell the developer that they miss to change the token value.

Reply:
Sure. I chose to test the value because I thought it would be better to have it as an optionnal feature rather than a mandatory element. I'm changing it anyway.